powered-by.html

```diff
@@ -32,6 +32,11 @@
         "logo": "addthis.png",
         "logoBgColor": "#ffffff",
         "description": "Apache Kafka is used at AddThis to collect events generated by our data network and broker that data to our analytics clusters and real-time web analytics platform."
+    }, {
+        "link": "http://www.amadeus.com",
+        "logo": "amadeus.jpg",
+        "logoBgColor": "#ffffff",
+        "description": "Apache Kafka is used for both real time and batch data processing, and is the chosen event log technology for Amadeus micro service based streaming applications. Kafka is also used for operational use cases such as application logs collection. Regarding your last question we are already in advanced discussions with Confluent for commercial support, training & tools (such as Replicator). We also had a session last week regarding a commercial framework (e.g. partnership) related to streaming applications on the cloud, with two axes:1) Support of Kafka on Red Hat OpenShift. 2) Streaming application topology management & monitoring on OpenShift."
```

Comment:
About the `we also had a session ..` sentence, I feel it is timely related and may not be understandable for the long run? cc @derrickdoo